mlflow/utils/search_utils.py

```diff
@@ -442,8 +442,7 @@ def _invalid_statement_token_search_runs(cls, token):
             return False
         elif token.match(ttype=TokenType.Keyword, values=["AND"]):
             return False
-        else:
```

Comment:
comment: I prefer this style to the following style
```python
return not token.match(ttype=TokenType.Keyword, values=["AND"])
```

Reply:
Can we make the following change instead?

```diff
diff --git a/mlflow/utils/search_utils.py b/mlflow/utils/search_utils.py
index 08a6bf959..f514fbb05 100644
--- a/mlflow/utils/search_utils.py
+++ b/mlflow/utils/search_utils.py
@@ -436,14 +436,11 @@ class SearchUtils:
 
     @classmethod
     def _invalid_statement_token_search_runs(cls, token):
-        if isinstance(token, Comparison):
-            return False
-        elif token.is_whitespace:
-            return False
-        elif token.match(ttype=TokenType.Keyword, values=["AND"]):
-            return False
-        else:
-            return True
+        return not (
+            isinstance(token, Comparison)
+            or token.is_whitespace
+            or token.match(ttype=TokenType.Keyword, values=["AND"])
+        )
 
     @classmethod
     def _process_statement(cls, statement):
```

Reply:
Yes, it would work. Also, we can use `any` too.
Do you like your suggestion more than the current one?

Reply:
Yes, `any` sounds good.

Reply:
I think we should use `or` here to avoid evaluating `token.is_whitespace` or `token.match(ttype=TokenType.Keyword, values=["AND"])` when `isinstance(token, Comparison)` is True.

Reply:
I'll create a good first for this change.